docs/README.md

```diff
@@ -102,6 +102,61 @@ The block still doesn't have CSS Handles for its specific customization.
 All CSS Handles available for the Image block are the ones available for the `slider-layout` block. Take a look at the Customization section in the [**Slider Layout documentation**](https://vtex.io/docs/app/vtex.slider-layout).
 Note that the `image-slider` uses our `vtex.slider-layout` app, so all the CSS namespaces defined by it are also available for `image-slider`. Take a look at [Slider-Layout](https://vtex.io/docs/app/vtex.slider-layout).
 
+## Image Protocol Implementation
+
+In the `manifest.json` we added Session Client app and also the Image Protocol app in the dependency list:
+
+```json
+ "dependencies": {
+    "vtex.session-client": "1.x",
+    "vtex.image-protocol": "0.x"
+  }
+```
+The interface `ImageTypes.ts` is modified to have the isMobile and imageProtocolId props:
+
+```ts
+export interface ImageSchema {
+  isMobile?: boolean
+  imageProtocolId? : string
+  ...
+}
+```
+To be able to personalize a specific image the props defined in the interface are also exposed in the Image component: 
+
+```tsx
+const{
+  isMobile=false,
+  imageProtocolId='',
+  ...
+}=props
+``` 
+
+>**NOTE**: The prop imageProtocolId can be modified from the Site Editor.
```

Comment:
```suggestion
> ⚠️
> 
> The prop `imageProtocolId` can be modified via Site Editor.
```

Reply:
How can we modify the prop via Site Editor?

Reply:
If you go to the Site Editor and check the List of images, selecting any of the images you will see the field 'Image Protocol Id'
<img width="1359" alt="Captura de pantalla 2022-06-10 a las 12 10 56" src="https://user-images.githubusercontent.com/104071100/173044274-09025b46-9c50-4418-8671-ab064beca67f.png">